docs/reference/runtimes/python/python-reference.md

```diff
@@ -64,9 +67,16 @@ CMD [ "processor" ]
 Your function-configuration file (for example, **my-function.yaml** for the [example Dockerfile](#dockerfile)) must include the name of your handler function and Python runtime. For more information, see the [function-configuration reference](/docs/reference/function-configuration/function-configuration-reference.md). For example:
```

Comment:
it is not. just more common
it can be overridden when running the `processor` by specifying the function configuration full path `--config /path/to/some.yaml`